File: src/test/java/se/johannesdahlgren/aoc24/Day11Test.java
package se.johannesdahlgren.aoc24;

import static org.assertj.core.api.Assertions.assertThat;

import java.io.IOException;
import org.junit.jupiter.api.Test;

public class Day11Test {

  @Test
  void part1() throws Exception {
    long count = new Day11().processInput(25);
    assertThat(count).isEqualTo(186175L);
  }

  @Test
  void part2() throws Exception {
    long count = new Day11().processInput(75);
    assertThat(count).isEqualTo(220566831337810L);
  }
}

File: src/test/kotlin/se/johannesdahlgren/aoc22/Day11Test.kt
package se.johannesdahlgren.aoc22

import org.assertj.core.api.Assertions.assertThat
import org.junit.jupiter.api.Test

class Day11Test {
    @Test
    fun part1Example() {
        val day11 = Day11("day11Example")
        assertThat(day11.monkeyBusiness()).isEqualTo(10605L)

    }

    @Test
    fun part1() {
        val day11 = Day11("day11")
        assertThat(day11.monkeyBusiness()).isEqualTo(118674L)
    }
}